raiwidgets/tests/conftest.py

```diff
@@ -73,3 +73,28 @@ def create_rai_insights_object_regression():
                   skip_cat_limit_checks=True)
     ri.compute()
     return ri
+
+
+@pytest.fixture(scope='session')
+def create_rai_insights_object_multiclass_classification():
```

Comment:
nit: maybe shorten the name to:

create_rai_insights_multiclass_classification

I don't think object is needed in the name

Reply:
The fixture indeed returns an RAIInsights object. Not sure if there is need to shorten the fixture name unless it violates the linter rules.

Reply:
the other fixtures also carry the object. So keeping it the same,